kolibri/core/negotiation.py

```diff
@@ -1,21 +1,29 @@
-from rest_framework.negotiation import BaseContentNegotiation
+from rest_framework.negotiation import DefaultContentNegotiation
 from rest_framework.parsers import JSONParser
 from rest_framework.parsers import MultiPartParser
 from rest_framework.renderers import JSONRenderer
 
 
-class LimitContentNegotiation(BaseContentNegotiation):
+class LimitContentNegotiation(DefaultContentNegotiation):
     def select_parser(self, request, parsers):
         """
         Always return JSONParser unless a 'multipart/form-data' content type is sent
+        or this is for morango
         """
+
```

Comment:
Note, I think we can just remove the `select_parser` override here, and just default to the `DefaultContentNegotiation` parser strategy - it's only the renderer that really needs to be overridden for performance reasons.

Reply:
I can make this change